RecoEgamma/EgammaTools/src/HGCalShowerShapeHelper.cc

```diff
@@ -0,0 +1,287 @@
+#include "RecoEgamma/EgammaTools/interface/HGCalShowerShapeHelper.h"
+
+const double HGCalShowerShapeHelper::kLDWaferCellSize_ = 0.698;
+const double HGCalShowerShapeHelper::kHDWaferCellSize_ = 0.465;
+
+void HGCalShowerShapeHelper::initPerEvent(const edm::EventSetup &iSetup, const std::vector<reco::PFRecHit> &pfRecHits) {
+  edm::ESHandle<CaloGeometry> geom;
+  iSetup.get<CaloGeometryRecord>().get(geom);
+  recHitTools_.setGeometry(*(geom.product()));
```

Comment:
use esConsumes, ESHandle without token is deprecated: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#In_ED_module

Reply:
Thanks for the pointer. I will fix it.